### PR TITLE
[ONNX] Momentum setting in SyncBatchNorm forward (inference) pass.

### PR DESCRIPTION
**Pull request by spandantiwari**
This is a fix for a potential ONNX export issue with SyncBatchNorm where irrespective of the value of momentum, the value for momentum in ONNX BN node is always 0. The details are captured in https://github.com/pytorch/pytorch/issues/18525.

The fix in this PR for `SyncBatchNorm` is very similar to the fix that went in https://github.com/pytorch/pytorch/pull/18764 for `BatchNorm` (I think this site was just missed). 

Please note that there are no ONNX test points added for this, because SyncBatchNorm works exclusively with tensors on GPU and the ONNX test passes are CPU only. If there's a way to add a test point, please let me know.

### CODE REVIEW

**Comment by spandantiwari** [comment]
Hi @SsnL - you reviewed a very similar PR https://github.com/pytorch/pytorch/pull/18764 earlier. Could you please take a look at this as well. Thanks.

File: torch/nn/modules/batchnorm.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def _check_input_dim(self, input):
     def forward(self, input):
         self._check_input_dim(input)
 
-        # exponential_average_factor is self.momentum set to
+        # exponential_average_factor is set to self.momentum
```
Comment (spandantiwari) [inline torch/nn/modules/batchnorm.py]: Typo fix in existing comment.

**Comment by spandantiwari** [comment]
Test failure was unrelated and seems to have passed since then.

**Review by dzhulgakov — APPROVED** [review]
LGTM as it's identical to the BatchNorm one
